src/stats/distribution.rs

```diff
@@ -0,0 +1,100 @@
+use core::ops::Range;
+use regex::Regex;
+use std::collections::HashMap;
+
+fn levels(threshold: u64) -> Vec<Range<u64>> {
+    vec![
+        (0..threshold),
+        (threshold..10_000),
+        (10_000..20_000),
+        (20_000..50_000),
+        (50_000..100_000),
+        (100_000..250_000),
+        (250_000..500_000),
+        (500_000..1_000_000),
+        (1_000_000..5_000_000),
+        (5_000_000..10_000_000),
+        (10_000_000..25_000_000),
+        (25_000_000..50_000_000),
+        (50_000_000..32_000_000_000),
```

Comment:
just to nitpick, this is not exhaustive and there's no guarantee in Jormungandr itself that this value will not be reached